xbmc/video/VideoDatabase.cpp

```diff
@@ -193,6 +193,13 @@ void CVideoDatabase::CreateTables()
 
   CLog::Log(LOGINFO, "create uniqueid table");
   m_pDS->exec("CREATE TABLE uniqueid (uniqueid_id INTEGER PRIMARY KEY, media_id INTEGER, media_type TEXT, value TEXT, type TEXT)");
+
+  CLog::Log(LOGINFO, "create type table");
+  m_pDS->exec("CREATE TABLE type (type_id INTEGER PRIMARY KEY, name TEXT)");
```

Comment:
To not cause any name conflicts if joining tables it might be better to name the column type.
```suggestion
  m_pDS->exec("CREATE TABLE type (type_id INTEGER PRIMARY KEY, type TEXT)");
```

Reply:
I checked all other similar tables, like tag/actor/genre, all of them have the id column named like "xxx_id" and name column named with "name", so I chose follow this style.